packages/connector-types/jest.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Better write in `.ts`.
You can do it later.

Reference: packages/ui/jest.config.ts